google-cloudevent-types/src/main/java/com/google/events/cloud/audit/v1/AuditLog.java

```diff
@@ -0,0 +1,122 @@
+package com.google.events.cloud.audit.v1;
```

Comment:
Possibly file an issue to add copyright headers to the generated code?

Reply:
I'd like to do that, but there are some issues with manually adding it here – probably should be added to the generator. We should do this at a later point.